common/djangoapps/third_party_auth/tests/specs/test_google.py

```diff
@@ -53,7 +53,7 @@ def test_custom_form(self):
         """
         Use the Google provider to test the custom login/register form feature.
         """
-        # The pipeline starts by a user GETting /auth/login/google-oauth2/?auth_entry=custom1
+        # The pipeline starts by a user Getting /auth/login/google-oauth2/?auth_entry=custom1
```

Comment:
Oh jeez, no idea how this got through. I noticed the mistake earlier (that it's referring to the HTTP method and isn't a typo) and tried reverting the commit but guess I made a mistake when rewriting the history.